### ISSUE TITLE
Require issue with build version of ext/emmet

### ISSUE THREAD

**Issue by adamjimenez** [ajaxorg/ace#2800]
Require breaks when trying to load modules which are loaded by Emmet's own require module.

I get errors like

```
Uncaught Error: Script error for: resources
```

see this line (from: https://github.com/ajaxorg/ace-builds/blob/0982db4853e3c967756b83b70638b9761d7f801d/src/ext-emmet.js#L904):

```
define("ace/ext/emmet",["require","exports","module","ace/keyboard/hash_handler","ace/editor","ace/snippets","ace/range","resources","resources","range","tabStops","resources","utils","actions","ace/config","ace/config"], function(require, exports, module) {

```

I can fix this problem by manually removing this bit

```
"resources","resources","range","tabStops","resources","utils","actions"
```

Presumably there is something up with the build process.
Also not sure why "resources" was in there twice.

**Comment by minwe** [comment]
`ext/emmet` is out of date, can't use with the latest emmet (emmet API changed).

**Comment by umireon** [comment]
This will be fixed by #3147.
